routes/productRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const productController = require('../controller/productController')
+
+const router = express.Router()
+
+router.get('/',productController.productAll)
+router.get('/:id',productController.getProduct)
+router.post('/create',productController.createProduct)
```

Comment:
```suggestion
router.post('/',productController.createProduct)
```
- `[POST]` 라는 http method만으로도 데이터를 추가한다는 의미가 통하기 때문에 이용하지 않으셔도 됩니다!